examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+import torch
+from PIL import Image
+import numpy as np
+import argparse
+import os
+import subprocess
+
+#Use most upto date weights
+inception_v3 = models.inception_v3(weights=models.Inception_V3_Weights.DEFAULT, progress=True)
+
+# Download ImageNet labels
+#!curl -o imagenet_classes.txt https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt
+
+def parse_input_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--fp16",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Perform fp16 quantization on the model before running inference',
+    )
+
+    parser.add_argument(
+        "--image_dir",
+        required=False,
+        default="../resnet50/images",
+        help='Target DIR for images to infer. Default is ./images'
+    )
+
+    parser.add_argument(
+        "--batch",
+        required=False,
+        default=1,
+        help='Batch size of images per inference',
+        type=int
+    )
+
+    parser.add_argument(
+        "--top",
+        required=False,
+        default=1,
+        help='Show top K of inference results',
+        type=int
+    )
+
+    parser.add_argument(
+        "--QPS",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show inference result in Queries-Per-Second QPS instead of inference duration (milliseconds)',
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show verbose output',
+    )
+    
+    return parser.parse_args()
+
+def get_image_list_in_dir(dir):
+    proc = subprocess.run("ls",
+                          shell=True,
+                          stdout=subprocess.PIPE,
+                          cwd=dir)
+    fileList = proc.stdout.decode().split('\n')
+    fileList
+    return fileList
+
+def softmax(x):
+    """Compute softmax values for each sets of scores in x."""
+    e_x = np.exp(x - np.max(x))
+    return e_x / e_x.sum()
+
+def run_sample(session, categories, latency, inputs, topk, batch_size):
+    
+    io_binding = session.io_binding()
+    io_binding.bind_cpu_input('input', inputs.cpu().detach().numpy())
+    io_binding.bind_output('output')
+    start = time.time()
+    session.run_with_iobinding(io_binding)
+    latency.append(time.time() - start)
+    ort_outputs = io_binding.copy_outputs_to_cpu()[0]
+
+    #Get prediction for each item
+    for i in range(batch_size):
+        output = ort_outputs[i].copy().flatten()
+        output = softmax(output)  # this is optional
+        top5_catid = np.argsort(-output)[:topk]
+
+        for catid in top5_catid:
+            print(categories[catid], output[catid])
+
+def main():
+    flags = parse_input_args()
+
+    if flags.verbose:
+        print(flags)
+
+    if flags.verbose:
+        print("Reading in Imagenet classes")
+
+    # Read the categories
+    with open("./imagenet_classes.txt", "r") as f:
```

Comment:
Is this assumed to be available ?

Reply:
yes. Will restructure folder here to put images and image_classes.txt in an images directory shared between this and resnet50 examples

Reply:
Should we just pull this down/store here as well? Its shared between both examples

Reply:
I think this should exist with the dataset, all the commonly available models use the same class mappings when training on imagenet as far as I know

Reply:
Done